README.md

```diff
@@ -14,53 +16,31 @@ To check that the install worked after you're done, run the following command
 docker --version
 ```
 
-### 2. Install homebrew:
-
-If you're not sure, check if you have homebrew installed
-
-```
-homebrew --version
-```
-
-If you do not have it, install with the following command
+### 2. Install Xquartz:
 
+Visit the [official site](https://xquartz.org)
 
-```
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
-```
```

Comment:
Homebrew is the primary package manager for developers on Mac OS. There's no reason to omit this installation instruction to a class of computer-something freshmen. If a student doesn't have it yet, they should.

Plus, Docker (~5Gb) is WAY bigger than Homebrew (> 7x).